src/utils/error.ts

```diff
@@ -737,6 +737,10 @@ export class AuthClientErrorCode {
     code: 'invalid-recaptcha-enforcement-state',
     message: 'reCAPTCHA enforcement state must be either "OFF", "AUDIT" or "ENFORCE".'
   }
+  public static RECAPTCHA_NOT_ENABLED = {
+    code: 'racaptcha-not-enabled',
+    message: 'reCAPTCHA enforcement state must be either "AUDIT" or "ENFORCE" to enable account defender.'
+  }
```

Comment:
This error is not specific to account defender. Maybe change the error message to just "reCAPTCHA enterprise is not enabled."

Reply:
Good call! Fixed it.